### ISSUE TITLE
Retire

### ISSUE THREAD

**Issue by g1eny0ung** [pingcap/pd-web#20]
Seems that it's time to make `pd-web` retired because its replacement `tidb-dashboard` is approaching maturity gradually.

We can delete this repository to completely avoid developers' confusion about it.

What do you think? @sivagao @baurine @breeswish @andylokandy

**Comment by baurine** [comment]
Or maybe archive it?

**Comment by g1eny0ung** [comment]
> Or maybe archive it?

If the code here has no residual value (I personally think so), I prefer to delete.

**Comment by breeswish** [comment]
Yes, I think this repo can be retired. Most of the source codes are placed in TiDB Dashboard now.

**Comment by sivagao** [comment]
LGTM

Wenxuan <notifications@github.com>于2021年1月15日 周五上午11:50写道：

**Comment by g1eny0ung** [comment]
> Yes, I think this repo can be retired. Most of the source codes are placed in TiDB Dashboard now.

I will request to delete this repo later.

@baurine I forgot to mention this in the previous reply. In fact, there is another reason for deleting this repository. People often find this repo and ask about its development before, so we often guide them to the `tidb dashboard`.
